app/models/volunteer_application.rb

```diff
@@ -0,0 +1,21 @@
+class VolunteerApplication < ActiveRecord::Base
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
+  VALID_PHONE_REGEX = /[0-9]{10}/
```

Comment:
Also you'll be able to use this and the email validator in the `AdoptionApplication` model